package-lock.json

```diff
@@ -6433,15 +6433,13 @@
           "version": "1.0.0",
           "resolved": "https://registry.npmjs.org/balanced-match/-/balanced-match-1.0.0.tgz",
           "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c=",
-          "dev": true,
```

Comment:
I'm not sure if `package-lock.json` should be updated. Maybe @belcherj can bring us some light here?